doc/source/ray-air/examples/pytorch_resnet_batch_prediction.ipynb

```diff
@@ -0,0 +1,545 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "MNwt9bSG0hin"
+   },
+   "source": [
+    "# Performing GPU Batch Prediction on Images with a PyTorch Model\n",
+    "\n",
+    "In this example, we will introduce how to use the Ray AIR {class}`BatchPredictor <ray.train.batch_predictor.BatchPredictor>` for **large-scale batch inference with multiple GPU workers.**\n",
+    "\n",
+    "For this demo, we will use the [Imagenette](https://github.com/fastai/imagenette) dataset, which is a subset of Imagenet with 10 classes.\n",
+    "\n",
+    "First, we use {meth}`ray.data.read_images <ray.data.read_images>` to load the validation set from S3. Since the dataset is already structured with directory names as the labels, we can use the {class}`Partitioning <ray.data.datasource.Partitioning>` API to automatically extract image labels."
```

Comment:
Please also show the user (in comment) what the directory structure of s3 looks like

Reply:
Done！